4-WebApp-your-API/4-1-MyOrg/Client/Controllers/TodoListController.cs

```diff
@@ -20,7 +20,8 @@ public TodoListController(ITodoListService todoListService)
         [AuthorizeForScopes(ScopeKeySection = "TodoList:TodoListScopes")]
```

Comment:
>AuthorizeForScopes [](http://example.com/codeflow?start=9&length=18)

Noticed this late, but is this even needed in a  client app? i think we should just keep it in the API's controller